do.sh

```diff
@@ -946,6 +946,51 @@ usage() {
     exit 1
 }
 
+pylint() {
+    mkdir -p logs
+    py=.env/bin/python
+    log=logs/pylint.log
+    dir=${1:-.}
+    reformat=${2:-false}
+
+    ${py} -m black ${dir} --check > ${log} 2>&1
+    if grep -q "Oh no!" ${log};
+    then
+        sed 's/would reformat/Black formatting failed for/' ${log} 
+        if [ $reformat = true ]
+        then
+            ${py} -m black ${dir}
+        else
+            exit 1
+        fi
+    fi
+}
+
+golint() {
+    mkdir -p logs
+    log=logs/golint.log
+    dir=${1:-"."}
+    reformat=${2:-false}
+
+    GO111MODULE=on CGO_ENABLED=0 go install -v github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2
+    lintdir=$([ ${dir} = "." ] && echo "./..." || echo ${dir})
+    go mod tidy
+    $HOME/go/bin/golangci-lint run --disable gosimple --timeout 5m -v ${lintdir} > ${log} 2>&1
```

Comment:
two observations
1. first I think the way I am using the whole path for golangci-lint can we have a better way that we just write golangci-lint run
2. had to include --disable gosimple cause gofmt -s does not format some of the stuffs which golanci-lint wants it to be in proper format